hera_cal/abscal.py

```diff
@@ -969,10 +982,11 @@ def omni_abscal_arg_parser():
     a.add_argument("--TT_phs_max_iter", type=int, default=10, help="maximum number of iterations of TT_phs_cal allowed")
     a.add_argument("--TT_phs_conv_crit", type=float, default=1e-6, help="convergence criterion in Delta g / g for stopping iterative TT_phs_cal")
     a.add_argument("--max_dlst", default=0.005, type=float, help="maximum allowed LST difference in model rephasing, otherwies model is flagged.")
+    a.add_argument("--refant", default=None, type=int, help="antenna number integer to use as reference antenna.")
     return a
 
 
-def abscal_run(data_file, model_files, calfits_infile=None, verbose=True, overwrite=False, write_calfits=True,
+def abscal_run(data_file, model_files, refant=None, calfits_infile=None, verbose=True, overwrite=False, write_calfits=True,
```

Comment:
You've centered the antpos that lives inside AC, but I don't think you've centered `total_data_antpos` around the reference antenna. Is that a problem?

Reply:
Bumping this question.

Reply:
Bumping again.